test/fast/Unit tests/nvm_get_mirror

```diff
@@ -12,5 +12,17 @@ die () { echo "$@" ; exit 1; }
 ! nvm_get_mirror node foo || die 'unknown release type did not error'
 ! nvm_get_mirror iojs foo || die 'unknown release type did not error'
 
-[ "$(nvm_get_mirror node std)" = "${NVM_NODEJS_ORG_MIRROR}" ] || die 'node std did not yield NVM_NODEJS_ORG_MIRROR'
-[ "$(nvm_get_mirror iojs std)" = "${NVM_IOJS_ORG_MIRROR}" ] || die 'node std did not yield NVM_NODEJS_ORG_MIRROR'
+
+[ -z "$NVM_NODEJS_ORG_MIRROR" ] || die "MIRROR environment variables should no longer be exported"
```

Comment:
I think travis-ci might set these; we should probably explicitly unset them on line 6 above, before sourcing nvm.sh, just to ensure that prior environment setup can't interfere with the test?